discovery-provider/src/__init__.py

```diff
@@ -272,6 +272,8 @@ def configure_celery(flask_app, celery, test_config=None):
 
     # Initialize IPFS client for celery task context
     gateway_addrs = shared_config["ipfs"]["gateway_hosts"].split(',')
+    gateway_addrs.append(shared_config["discprov"]["user_metadata_service_url"])
+    logger.warning(gateway_addrs)
```

Comment:
@hareeshnagaraj we need a standardized way of doing logs. Like logger.warning(f' <filename> | <function> | <message>'). not blocking for this